site/app/libraries/Core.php

```diff
@@ -244,7 +244,7 @@ public function loadForum() {
             throw new \Exception("Need to load the config before we can create a forum instance.");
         }
 
-        $this->forum = new Forum($this);
+        //$this->forum = new Forum($this);
```

Comment:
Should this be commented out? There are other mentions of a forum member in core which all seems unused, but I think that should be removed in its own PR instead of here.

Reply:
I'll remove from this PR and make a new PR for it. We discussed it should be turned off from being created in Core since it is never used. We are still keeping the code so a refactor can be done in the future.